src/app.scss

```diff
@@ -1,3 +1,7 @@
+@import 'carbon-components/scss/globals/scss/vendor/@carbon/type/scss/font-family.scss';
+@import 'carbon-components/scss/globals/scss/vendor/@carbon/layout/scss/breakpoint.scss';
+@import 'carbon-components/scss/globals/scss/typography.scss';
+@import 'carbon-components/scss/globals/scss/vars.scss';
 @import './components/TutorialHeader/tutorial-header.scss';
-@import './content/LandingPage/landing-page.scss';
-@import './content/RepoPage/repo-page.scss';
+@import './content/LandingPage/_landing-page.scss';
+@import './content/RepoPage/_repo-page.scss';
```

Comment:
These imports should not be changed..

Reply:
Your landing page background image is not rending properly and this may the cause.